### ISSUE TITLE
How to apply the Mermaid bundled netural theme?

### ISSUE THREAD

**Issue by alexwall** [vstirbu/vscode-mermaid-preview#75]
Mermaid Preview `0.12.0`

How to apply the `natural` theme?

This is what I've tried, but it doesn't work

```json
{
    "mermaid": {
        "theme": "neutral",
        "themeCSS": ""
    }
}
```

<img width="414" alt="image" src="https://user-images.githubusercontent.com/316469/58010662-15070780-7af1-11e9-8426-f5208c2c2339.png">

**Comment by vstirbu** [comment]
Use this for now:

```json
{
  "mermaid": {
    "theme": null,
    "themeCSS": "the content of the neutral theme from Mermaid",
  }
}
```

Would it be useful to provide a configuration flag for the light theme to choose between `forest` and `neutral`? Personally, I prefer `forest` but that seems not suit everybody ;)

**Comment by alexwall** [comment]
I guess it‘s better to provide a list of available themes.  

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by vstirbu** [comment]
Available in 1.1.1.
